Documentation/operations/system_requirements.rst

```diff
@@ -82,7 +82,7 @@ RancherOS_                 >= 1.5.5
 .. _RancherOS: https://rancher.com/rancher-os/
 
 .. [#centos_foot] CentOS 7 requires a third-party kernel provided by `ElRepo <http://elrepo.org/tiki/tiki-index.php>`_
-    whereas CentOS 8 ships with a supported kernel.
+    whereas CentOS 8 ships with a supported kernel. Cilium features that require large BPF programs (eg. VTEP integration) may not work on CentOS 7, as the ``bpftool`` version shipped with Cilium can't probe for ``HAVE_LARGE_INSN_LIMIT`` support due to an incompatibility with the ``glibc`` version on CentOS 7.
```

Comment:
Maybe we can just make a more generic message and improve the base kernel requirements statements below.

```suggestion
    whereas CentOS 8 ships with a supported kernel. Note that some more advanced features may not be available on Centos 7 even with a third-party kernel. For full details on which kernel config options must be enabled in order to use various features, see the section on :ref:`admin_kernel_version` requirements below.
```